### PR TITLE
Feature/aa 303 add notifications for agency management events

### PR DESCRIPTION
**Pull request by AlexeyShok**
Added notification for master agent for changing the status of the agency's activity

### CODE REVIEW

File: Api/Infrastructure/ServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -269,12 +269,16 @@ public static class ServiceCollectionExtensions
...
+                options.AgencyActivityChangedTemplateId = agencyActivityChangedId;
+            });
             #endregion
-            
+
             #region tag processing options
-            
-            var tagProcessingOptions = vaultClient.Get(configuration["Edo:TagProcessing:Options"]).GetAwaiter().GetResult();
+
+var tagProcessingOptions = vaultClient.Get(configuration["Edo:TagProcessing:Options"]).GetAwaiter().GetResult();
```
Comment (oleg1std) [inline Api/Infrastructure/ServiceCollectionExtensions.cs:281]: formattin

Comment (AlexeyShok) [reply Api/Infrastructure/ServiceCollectionExtensions.cs]: Fixed

File: Api/Infrastructure/Options/AgencyManagementMailOptions.cs [hunk 2]
```diff
@@ -0,0 +1,7 @@
+﻿namespace HappyTravel.Edo.Api.Infrastructure.Options
+{
+    public class AgencyManagementMailOptions
```
Comment (kirillta) [inline Api/Infrastructure/Options/AgencyManagementMailOptions.cs:3]: Probably we didn't call it that way, but I think the name should be _Mailing_ options

File: HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs [hunk 3]
```diff
@@ -20,7 +20,7 @@ public enum NotificationTypes
         ChildAgencyInvitation = 12,
         AgentSuccessfulRegistration = 13,
         ChildAgencySuccessfulRegistration = 14,
-        AgencyManagement = 15,
+        AgencyActivityChanged = 15,
```
Comment (kirillta) [inline HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs:23]: Why?

File: Api/Infrastructure/Options/AgencyManagementMailOptions.cs (hunk 2 above)
Comment (AlexeyShok) [reply Api/Infrastructure/Options/AgencyManagementMailOptions.cs:3]: Renamed

File: HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs (hunk 3 above)
Comment (AlexeyShok) [reply HappyTravel.Edo.Notifications/Enums/NotificationTypes.cs:23]: We have our own email template for each event. Therefore, each event must have its own value in the enumeration. For counterparty, for example, there are already two such events: changing the verification status and activating / deactivating counterparty. So I renamed this notification type to be more specific.

**Review by kirillta — APPROVED** [review]

**Pull request merged**